docker-entrypoint.sh

```diff
@@ -23,6 +23,6 @@ if [ "$MY_GID" != "$VOLUME_GID" ] || [ "$MY_UID" != "$VOLUME_UID" ]; then
     cmd="cd $PWD; $@"
     su btrix -c "$cmd"
```

Comment:
A quick change to also 
```suggestion
su btrix -c 'cd $PWD; "$@"' -- argv0-ignore "$@"
```

Based on https://unix.stackexchange.com/questions/479587/first-argument-passed-to-wrapper-script-is-ignored/479645#479645